pkg/result/processors/fixer.go

```diff
@@ -197,16 +197,14 @@ func (p Fixer) printStat() {
 	p.sw.PrintStages()
 }
 
-func skipTextEditWithoutPosition(issue *result.Issue) bool {
+func skipNoTextEdit(issue *result.Issue) bool {
 	var onlyMessage int
 	var count int
 	for _, sf := range issue.SuggestedFixes {
-		for _, edit := range sf.TextEdits {
-			count++
-			if edit.Pos == token.NoPos && edit.End == token.NoPos {
-				onlyMessage++
-			}
+		if len(sf.TextEdits) == 0 {
+			onlyMessage++
 		}
+		count++
 	}
 
 	return count == onlyMessage
```

Comment:
After removing the inner `for` loop, the `count` equals `len(issue.SuggestedFixes)`

```suggestion
	return len(issue.SuggestedFixes) == onlyMessage
```